applications/tari_console_wallet/src/cli.rs

```diff
@@ -39,7 +39,7 @@ pub(crate) struct Cli {
     /// Supply the password for the console wallet. It's very bad security practice to provide the password on the
     /// command line, since it's visible using `ps ax` from anywhere on the system, so always use the env var where
     /// possible.
-    #[clap(long)] // , env = "TARI_WALLET_PASSWORD")]
+    #[clap(long, env = "TARI_WALLET_PASSWORD", hide_env_values = true)]
```

Comment:
How about to hide it completely with `#[clap(hide = true)]` ?

Reply:
I think `hide_env_values` is correct here